### ISSUE TITLE
[Feature] Short Form Title (50 chars or less.)

### ISSUE THREAD

**Issue by jeremymeyers** [stashapp/stash#3860]
**Is your feature request related to a problem? Please describe.**
I have folders of scenes that I sometimes want to be in the mix of my collection but not always. For example, specific kink content or behind the scenes content. 

**Describe the solution you'd like**
I would like the ability to add these folders to the "library" list of my configuration and then have an additional checkbox for "show in collection".

When unchecked (checked by default) This box would prevent scenes/performers/studios from displaying within the interface or search results. It would not make any changes to the database or metadata for any of the associated entities. 

**Describe alternatives you've considered**
I'm not sure if this would be any easier, but perhaps this could be partially be done with the addition of a filter criteria based on the path of the file (e. g. PATH DOES NOT CONTAIN "feet") or whatever. 

**Additional context**

**Comment by tthe3d** [comment]
Agreed this would be a nice feature. Also another solution would be to show/hide tagged files. For example today I don't wanna browse "feet" , each file that has the tag "feet" is therefore not shown in "Scenes" . Keep up the good work 👯 

EDIT: I just found out the tag feature is already implemented. 👍
